src/main/java/com/github/mob41/blapi/pkt/auth/AuthPayload.java

```diff
@@ -34,6 +34,35 @@
 
     private final byte[] data;
 
+    public AuthPayload() {
+        data = new byte[0x50];
+
+        data[0x04] = 0x31;
+        data[0x05] = 0x31;
+        data[0x06] = 0x31;
+        data[0x07] = 0x31;
+        data[0x08] = 0x31;
+        data[0x09] = 0x31;
+        data[0x0a] = 0x31;
+        data[0x0b] = 0x31;
+        data[0x0c] = 0x31;
+        data[0x0d] = 0x31;
+        data[0x0e] = 0x31;
+        data[0x0f] = 0x31;
+        data[0x10] = 0x31;
+        data[0x11] = 0x31;
+        data[0x12] = 0x31;
+        data[0x1e] = 0x01;
+        data[0x2d] = 0x01;
+        data[0x30] = (byte) 'T';
+        data[0x31] = (byte) 'e';
+        data[0x32] = (byte) 's';
+        data[0x33] = (byte) 't';
+        data[0x34] = (byte) ' ';
+        data[0x35] = (byte) ' ';
+        data[0x36] = (byte) '1';
+    }
+
     public AuthPayload(byte[] devId, byte[] devName) {

```

Comment:
TODO: Below it is for custom devId and devName. Uncomment the code and remove the existing code.

Reply:
It looks as this is not needed. When comparing to the pyhton impl, it is not used, just the 'Test 1'.